docs/release.md

```diff
@@ -0,0 +1,14 @@
+# Release process
+
+We use semantic versioning for `obstctl` in format `vX.Y.Z`.
```

Comment:
```suggestion
We use semantic versioning for `obsctl` in format `vX.Y.Z`.
```

Reply:
Funnily enough, 'obst' means 'fruit' in German :banana: :apple: :grapes:, but we cannot control fruits with CLI (yet).